openapi.yaml

```
@@ -2165,6 +2176,631 @@ paths:
       security:
         - oscal_auth:
             - read:parties
+  /plan-of-action-and-milestones:
+    get:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Returns all OSCAL plan of action and milestones
+      operationId: getPoam
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/OSCALPoam"
+      security:
+        - oscal_auth:
+            - read:poams
+    post:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Adds a new OSCAL plan of action and milestone
```

Should `milestone` always be plural? That is how it appears on the [spec ](https://pages.nist.gov/OSCAL/reference/latest/plan-of-action-and-milestones/json-reference/)
```suggestion
      summary: Adds a new OSCAL plan of action and milestones
```